google/cloud/compute/v1/BUILD.bazel

```diff
@@ -3,64 +3,25 @@
 # repository.
 load(
     "//rules_gapic:disco_to_proto.bzl",
+    "gapic_yaml_from_disco",
+    "grpc_service_config_from_disco",
     "proto_from_disco",
 )
 
 proto_from_disco(
-    name = "compute_small",
-    src = "//:src/test/resources/compute.v1.small.json",
+    name = "compute",
```

Comment:
Would it be clearer to also call this `computesmall`? I had to look at the file to see the comment at the top that this is a demo file. The more descriptive name may help.